### PR TITLE
Sync develop branch changes since 2/16 to hdf5_1_14 branch

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

**Review by byrnHDF — APPROVED** [review]

File: release_docs/INSTALL_CMake.txt [hunk 1]
```diff
@@ -213,7 +213,7 @@ Notes: This short set of instructions is written for users who want to
            cmake -G "Visual Studio 16 2019" -A Win32 -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=OFF -DBUILD_TESTING:BOOL=ON -DHDF5_BUILD_TOOLS:BOOL=ON ..\hdf5-1.14."X"
 
          On Windows 64 bit
-           cmake -G "Visual Studio 16 2019 Win64" -A x64 -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=OFF -DBUILD_TESTING:BOOL=ON -DHDF5_BUILD_TOOLS:BOOL=ON ..\hdf5-1.14."X"
+           cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=OFF -DBUILD_TESTING:BOOL=ON -DHDF5_BUILD_TOOLS:BOOL=ON ..\hdf5-1.15."X"
```
Comment (jhendersonHDF) [inline release_docs/INSTALL_CMake.txt]: 1.14

File: release_docs/INSTALL_Auto.txt [hunk 2]
```diff
@@ -401,8 +350,10 @@ CONTENTS
             --enable-hl, --enable-cxx, --enable-fortran, --enable-java
 
 
-4.3.11. Backward compatibility
-        The 1.14 version of the HDF5 library can be configured to operate
+   3.11. Backward compatibility
+        The 1.15 version of the HDF5 library can be configured to operate
```
Comment (jhendersonHDF) [inline release_docs/INSTALL_Auto.txt]: Should this sentence be in the 1.14 branch?

Comment (byrnHDF) [reply release_docs/INSTALL_Auto.txt]: Of course not it should be 1.14 and the next two lines for 1.14 api should be removed

Comment (byrnHDF) [reply release_docs/INSTALL_Auto.txt]: Better make sure there are no other 1.15 refs in the PR!

Comment (lrknox) [reply release_docs/INSTALL_Auto.txt]: There are 3 more of them in HDF5Examples/CMakeUserPresets.json lines 22 - 27:

      "environment": {
        "HDF5_ROOT": "${sourceParentDir}/temp/HDF_Group/HDF5/1.15.0",
        "HDF5_PLUGIN_PATH": "${sourceParentDir}/temp/HDF_Group/HDF5/1.15.0/lib/plugin"
      },
      "cacheVariables": {
        "HDF5_ROOT": {"type": "STRING", "value": "${sourceParentDir}/temp/HDF_Group/HDF5/1.15.0"},

They were already there in the hdf5_1_14 branch.  I'll change them to 1.14.4, but @byrnHDF  do we need to add the file/lines to h5vers?

**Review by jhendersonHDF — APPROVED** [review]

**Review by byrnHDF — APPROVED** [review]

**Pull request merged**